dtcli/scripts/dt.py

```diff
@@ -11,7 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import os
+import sys
+sys.path.append(os.getcwd())

```

Comment:
I know why it's here, but I also think that it shouldn't have to be - can we investigate another method of having a reliable local development, while not polluting the code?